tests/plugins/example/override_comments.py

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (C) 2020 Satoru SATOH
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Can you please stay consistent with other license headers, even in the plugin example code?

Reply:
Thanks! I forgot to fix that. Fixed in b49e67f.